mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/social/SocialCardTest.kt

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.sdp.mobile.test.ui.social
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import ch.epfl.sdp.mobile.application.Profile.Color
+import ch.epfl.sdp.mobile.ui.social.Person
+import ch.epfl.sdp.mobile.ui.social.PersonCard
+import org.junit.Rule
+import org.junit.Test
+
+class SocialCardTest {
+  @get:Rule val rule = createComposeRule()
+
+  private class SnapshotFriendCard : Person {
```

Comment:
Nitpick (minor) : the term `Snapshot` is used when the state implementation uses the compose snapshot system, aka when it's using some `mutableState`, `mutableStateListOf` etc. under-the-hood (a great but unnecessary reading on this is https://dev.to/zachklipp/introduction-to-the-compose-snapshot-system-19cn). Therefore I'd suggest naming this object something like `FakeFriendCard` instead.

Reply:
Fixed in the commit https://github.com/epfl-SDP/android/pull/91/commits/b65986a76a6bda2522d0191d44e6fde3f25af30e